react.js

```diff
@@ -1,25 +1,5 @@
 // This config just adds support for React and es6 parsing
 
 module.exports = {
-  extends: 'zanata',
-
-  ecmaFeatures: {
-    jsx: true
-  },
-
-  env: {
-    node: false
-  },
-
-  plugins: [
-    'react'
-  ],
-
-  // Use the babel parser to handle es6 and jsx properly
-  parser: 'babel-eslint',
-
-  rules: {
-    'react/jsx-uses-react': 2,
-    'react/jsx-uses-vars': 2
-  }
```

Comment:
On second thought, the ability to use the same linting file for application code as well as build-related scripts does make things a lot simpler, particularly with getting IDEs and text editors to use the correct config.

I tested this and there is a webpack build error when trying to build the app using a node-only API, so it should be safe to rely on webpack to catch any accidental use of node APIs. The test was adding:

```javascript
import { readFile } from 'fs'
console.log(readFile)
```

and it generated an error like

```bash
ERROR in ./actions/phrases.js
Module not found: Error: Cannot resolve module 'fs' in /home/damason/src/zanata/zanata-spa/app/redux-app/actions
 @ ./actions/phrases.js 32:10-23
```